{
  "uid" : "cfccffbf771139f6",
  "name" : "Attempt to revoke a non-existent certificate",
  "fullName" : "Certificate lifecycle management via TPC_Web: Attempt to revoke a non-existent certificate",
  "historyId" : "6ec41c33a3afea392e463eed55ca005e",
  "time" : {
    "start" : 1746705351126,
    "stop" : 1746705351697,
    "duration" : 571
  },
  "status" : "failed",
  "statusMessage" : "AssertionError: STEP: When the admin attempts to <revoke> a non-existent certificate \"NONEXISTENT\"\n",
  "statusTrace" : "  File \"/home/runner/work/trustpoint/trustpoint/.venv/lib/python3.12/site-packages/behave/model.py\", line 1909, in run\n    match.run(runner.context)\n  File \"/home/runner/work/trustpoint/trustpoint/.venv/lib/python3.12/site-packages/behave/matchers.py\", line 104, in run\n    self.func(context, *args, **kwargs)\n  File \"trustpoint/features/steps/r_003_steps.py\", line 110, in step_attempt_nonexistent_certificate\n    raise AssertionError(msg)\n",
  "flaky" : false,
  "newFailed" : false,
  "beforeStages" : [ ],
  "testStage" : {
    "status" : "failed",
    "statusMessage" : "AssertionError: STEP: When the admin attempts to <revoke> a non-existent certificate \"NONEXISTENT\"\n",
    "statusTrace" : "  File \"/home/runner/work/trustpoint/trustpoint/.venv/lib/python3.12/site-packages/behave/model.py\", line 1909, in run\n    match.run(runner.context)\n  File \"/home/runner/work/trustpoint/trustpoint/.venv/lib/python3.12/site-packages/behave/matchers.py\", line 104, in run\n    self.func(context, *args, **kwargs)\n  File \"trustpoint/features/steps/r_003_steps.py\", line 110, in step_attempt_nonexistent_certificate\n    raise AssertionError(msg)\n",
    "steps" : [ {
      "name" : "Given the admin user is logged into TPC_Web",
      "time" : {
        "start" : 1746705351128,
        "stop" : 1746705351669,
        "duration" : 541
      },
      "status" : "passed",
      "steps" : [ ],
      "attachments" : [ ],
      "parameters" : [ ],
      "stepsCount" : 0,
      "attachmentsCount" : 0,
      "shouldDisplayMessage" : false,
      "hasContent" : false
    }, {
      "name" : "When the admin attempts to <revoke> a non-existent certificate \"NONEXISTENT\"",
      "time" : {
        "start" : 1746705351669,
        "stop" : 1746705351669,
        "duration" : 0
      },
      "status" : "failed",
      "statusMessage" : "AssertionError: STEP: When the admin attempts to <revoke> a non-existent certificate \"NONEXISTENT\"\n",
      "statusTrace" : "  File \"/home/runner/work/trustpoint/trustpoint/.venv/lib/python3.12/site-packages/behave/model.py\", line 1909, in run\n    match.run(runner.context)\n  File \"/home/runner/work/trustpoint/trustpoint/.venv/lib/python3.12/site-packages/behave/matchers.py\", line 104, in run\n    self.func(context, *args, **kwargs)\n  File \"trustpoint/features/steps/r_003_steps.py\", line 110, in step_attempt_nonexistent_certificate\n    raise AssertionError(msg)\n",
      "steps" : [ ],
      "attachments" : [ ],
      "parameters" : [ ],
      "stepsCount" : 0,
      "attachmentsCount" : 0,
      "shouldDisplayMessage" : true,
      "hasContent" : true
    }, {
      "name" : "Then the system should display an error message stating \"Certificate not found\"",
      "time" : {
        "start" : 1746705351697,
        "stop" : 1746705351697,
        "duration" : 0
      },
      "status" : "skipped",
      "steps" : [ ],
      "attachments" : [ ],
      "parameters" : [ ],
      "stepsCount" : 0,
      "attachmentsCount" : 0,
      "shouldDisplayMessage" : false,
      "hasContent" : false
    } ],
    "attachments" : [ ],
    "parameters" : [ ],
    "stepsCount" : 3,
    "attachmentsCount" : 0,
    "shouldDisplayMessage" : false,
    "hasContent" : true
  },
  "afterStages" : [ ],
  "labels" : [ {
    "name" : "severity",
    "value" : "normal"
  }, {
    "name" : "feature",
    "value" : "Certificate lifecycle management via TPC_Web"
  }, {
    "name" : "framework",
    "value" : "behave"
  }, {
    "name" : "language",
    "value" : "cpython3"
  }, {
    "name" : "resultFormat",
    "value" : "allure2"
  } ],
  "parameters" : [ ],
  "links" : [ ],
  "hidden" : false,
  "retry" : false,
  "extra" : {
    "severity" : "normal",
    "retries" : [ ],
    "categories" : [ {
      "name" : "Product defects",
      "matchedStatuses" : [ ],
      "flaky" : false
    } ],
    "history" : {
      "statistic" : {
        "failed" : 2,
        "broken" : 0,
        "skipped" : 0,
        "passed" : 0,
        "unknown" : 0,
        "total" : 2
      },
      "items" : [ {
        "uid" : "75c27fa4415f33e9",
        "reportUrl" : "https://Trustpoint-Project.github.io/trustpoint/16/#testresult/75c27fa4415f33e9",
        "status" : "failed",
        "statusDetails" : "AssertionError: STEP: When the admin attempts to <revoke> a non-existent certificate \"NONEXISTENT\"\n",
        "time" : {
          "start" : 1746616902251,
          "stop" : 1746616902831,
          "duration" : 580
        }
      } ]
    },
    "tags" : [ ]
  },
  "source" : "cfccffbf771139f6.json",
  "parameterValues" : [ ]
}